docs/README.md

```diff
@@ -24,6 +24,16 @@ Theme is copied from Daml Docs theme.
 
 `Node` and `yarn` must be installed on your machine.
 
+### Sass
+
+The `Node` version of `sass` must be installed: `gem install sass`
```

Comment:
What's `gem` ?

Reply:
it's a package manager for ruby

Reply:
`gem` looks to be a ruby package manager. Shouldn't the `node` version be installed via `npm` ?